docs/DeveloperGuide.md

```diff
@@ -26,8 +26,8 @@ enhancements.
   * [3.9. Bus at bus stop finder (`/bus` Feature)](#39-bus-at-bus-stop-finder-bus-feature)
   * [3.10. Performing similarity checks](#310-performing-similarity-checks)
   * [3.11. Displaying most searched bus stop](#311-displaying-most-searched-bus-stop-on-start-up)
-    + [3.11.1 Resetting all search frequencies](#3111-resetting-search-frequencies)
-- [4. Appendix A: Product scope](#4-appendix-a-product-scope)
+    + [3.11.1 Resetting all search frequencies](#3111-resetting-search-frequencies-of-bus-stops-reset-feature)
+- [4. Appendix I: Requirements](#4-appendix-i-requirements)
```

Comment:
Remember to add 3.12!

Reply:
Alright, so 3.11.1 should be 3.12 , is that right?